packages/web-components/src/components/va-additional-info/va-additional-info.css

```diff
@@ -2,7 +2,8 @@
 :host {
   display: block;
 }
-a[aria-expanded='true'] ~ #info {
+
+:host([disable-border='false']) a[aria-expanded='true'] ~ #info {
```

Comment:
This is closer, but what about when `disable-border` isn't passed at all?

![image](https://user-images.githubusercontent.com/2008881/159557799-2635fe17-394c-4173-8bda-87dcc4559970.png)

Reply:
@bkjohnson updated so it will only change the css when the prop is true